RadarSDK/RadarLocationManager.m

```diff
@@ -94,6 +101,43 @@ - (instancetype)init {
         _lowPowerLocationManager.distanceFilter = 3000;
         _lowPowerLocationManager.allowsBackgroundLocationUpdates = [RadarUtils locationBackgroundMode];
 
+        if ([RadarSettings useLocationMetadata]) { 
```

Comment:
I think this chunk should not be called here as the `RadarLocationManager` is actually initialized before our call to `getConfigForUsage`.  I don't think server-side configurations of `useLocationMetaData` will be applied. I think we should move it to `updateTracking`